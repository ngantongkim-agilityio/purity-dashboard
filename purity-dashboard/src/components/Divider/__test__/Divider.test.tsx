import { render } from '@testing-library/react';

// Components
import { Divider } from '..';

describe('Divider component', () => {
  it('Should render Divider component with default props', () => {
    const { container } = render(<Divider />);

    expect(container).toMatchSnapshot();
  });
});
