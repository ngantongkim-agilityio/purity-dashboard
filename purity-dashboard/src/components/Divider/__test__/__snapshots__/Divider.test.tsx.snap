// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider component Should render Divider component with default props 1`] = `
<div>
  <hr
    class="w-full h-divider shrink-0 bg-divider border-none"
    role="separator"
    variant="default"
  />
</div>
`;
